src/component-queries/DiseaseCellLines.tsx

```diff
@@ -63,43 +28,8 @@ const groupLines = (
     return cellLines.reduce((acc, cellLine) => {
         const { disease } = cellLine.node.frontmatter;
         const diseaseName = disease.frontmatter.name;
-        const cellLineData: UnpackedDiseaseCellLine = {
-            ...cellLine.node.frontmatter,
-            path: cellLine.node.fields.slug,
-            diseaseGene: null,
-            parentalLine: null,
-            key: cellLine.node.id,
-        };
```

Comment:
wrote a full data conversion function instead of this partial unpacking so all the terms in javascript are camel case